src/blocks/models.py

```diff
@@ -144,6 +144,59 @@ class Meta:
 # CONTENT BLOCKTYPES
 
 
+class CountdownBlock(blocks.StructBlock):
+    id = blocks.CharBlock()
```

Comment:
It would be good with a help text on id that explains what it does so that a user understands why it is needed and what to fill in. I assume it's to be able to have multiple countdowns on the same page?

Reply:
The original intention was just to have a unique handle to the element so I can reference it from javascript. Being able to reuse to ID to duplicate a counter was a nice side-effect, but come to think of it, it doesnt really make sense to specify the same ID in two different blocks since the other fields in the last block will overwrite both blocks anyway. 

Ill remove the ID field and think of something else.

Reply:
Fixed it with some clever javascript. If we group the script tag with the countdown, we can save the current script tag when loading, and then refer back to it to find the countdown in the parent tag.